index.html

```diff
@@ -1828,32 +1829,60 @@ <h1>Bindings</h1>
 	<section id="bindings-https">
 		<h2>HTTP(S) Binding</h2>
 		<p>This section defines a <a>DID resolver</a> <a>binding</a> which exposes the
-		<a>DID resolution</a> and/or <a>DID URL dereferencing</a> functions (including all
-		resolution options and output data) via an HTTP(S) endpoint. See <a href="#resolver-architectures"></a>.</p>
+			<a>DID resolution</a> and/or <a>DID URL dereferencing</a> functions (including all
+			resolution/dereferencing options and metadata) via an HTTP(S) endpoint. See <a href="#resolver-architectures"></a>.</p>
+
+		<p>The HTTP(S) binding is a <a>remote binding</a>. It requires a known HTTP(S) URL where a remote
```

Comment:
Wondering if we need to say that HTTPS can also be used locally. HTTPS does not mean always connecting to a remote DID resolver